lib/utils/rootrepositoryutils.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const chalk = require( 'chalk' );
+
+const ROOT_REPOSITORY_SUFFIX = '[ROOT REPOSITORY]';
+
+const ROOT_REPOSITORY_SUPPORTED_COMMANDS = [
+	'checkout',
+	// 'close',
```

Comment:
I did it this way because I wanted to clearly show which commands are unsupported, while also making this a whitelist instead of a blacklist, so that any future potential commands would have to be manually approved as supported here. On second thought though, perhaps this does not make much sense, as we'd have to update this list regardless when adding new commands.

I also agree that this approach is not ideal from the source of truth perspective. Thus, I've updated it to be a blacklist that uses imported names, instead of hard-coded ones.

One caveat about this approach though: `require`'ing of some commands results in an error due to circular `require()` chain. It just so happens that in case of blacklist we currently need, neither `close` nor `save` commands cause this issue.